### DISCUSSION TITLE
navigation.expand & hide:   - navigation

### DISCUSSION THREAD

**Discussion by PieDelDesVolcans** [squidfunk/mkdocs-material#7265] [Q&A]
Bonjour,

I'm trying to setup the following usage of the tool :

my_dir1\
  - my_dir1.1\
     - myfirstfile.md
     - mysecondfile.md

my_dir2\
  - afile.md
  - anotherfile.md
 
 I do would like to use the following behavior in my md files (just open the target file in full screen to display the information)
 
 ---
hide:
  - navigation
  - toc
---

So starting with the configuration : 

theme:
  features:
    - navigation.expand
   
if I setup in the md file the following : 
 ---
hide:
  - navigation
  - toc
---

Then : 
For directory my_dir2 : files afile.md & anotherfile.md html links are created for both files
But for directory my_dir1 : files myfirstfile.md & mysecondfile.md aren't created and only the first file is linked as html
Even if the files aren't displayed in the left panel they apperas in the next section at the bottom of the page.

If I remove the "navigation.expand" then :
For directory my_dir2 : only a link to afile.html appears
Same behavior for my_dir1

However : 
If I add in my_dir2 a empty mdfile starting with letter a ( first in alphabetical order) then links for a, myfirstfile & mysecondfile are displayed.

Would it be possible to use the combination of : not using "navigation.expand" & "hide navigation" ?

Thanks

**Comment by kamilkrzyskow** [comment]
Hello @PieDelDesVolcans,
after reading the post twice, I'm stil unsure what the issue is.😞What is the end result that you want to achieve? 
A [minimal reproduction](https://squidfunk.github.io/mkdocs-material/guides/creating-a-reproduction/#creating-a-reproduction) could help you provide a better example.

`navigation.expand` only opens up the sections in the navigation tree.
`hide: navigation` only hides the **display** of the navigation tree, it doesn't remove the page from the navigation nor does it remove the navigation tree on the page. The navigation simply is not displayed on the rendered page, but is still in the HTML code.

So neither of those options should affect relations or link creation to other files.

**Reply by PieDelDesVolcans** [reply]
@kamilkrzyskow,

Thanks for your answer. Many tests performed locally and the issue was between the laptop screen and the chair...

Have a nice week-end !
